OGR Source

SoerenHoffstedt opened the pull request:
Added OGR Source Operator and renamed old operator to OGR Raw Source Operator.
Added source listing of OGR datasets in user service. The functionality of that is in ogr_source_datasets.
Test queries and data are included.

michaelmattig commented on src/CMakeLists.txt:
```diff
@@ -92,7 +92,9 @@ add_library(mapping_core_base_lib
-        raster/opencl.cpp)
+        raster/opencl.cpp
+        util/ogr_source_datasets.cpp
+        util/ogr_source_datasets.h)
```
redundant include?

michaelmattig commented on test/systemtests/data/ogr_files/stationsWithCoordinates.csv:
```diff
@@ -0,0 +1,339 @@
+X,Y,Station_ID,type
```
where does this file come from? License?

SoerenHoffstedt replied:
I got the files (including the allStations files) from Johannes. @jdroenner do you know the license?

jdroenner replied:
I guess we should not use non public data for tests... Maybe we should use something else.

http://www.naturalearthdata.com/downloads/10m-cultural-vectors/ <-- Natural Earth is free: http://www.naturalearthdata.com/about/terms-of-use/

SoerenHoffstedt replied:
I removed the files and pushed new tests with files from Natural Earth.

SoerenHoffstedt commented:
I added the variable to `mapping_test.conf` and updated the file to be in TOML structure. I also adjusted the paths of other variables to fit the location of the executable.
I also found a bug in Configuration not loading configuration files set in an environment variable.
